packages/core/admin/shared/contracts/admin.ts

```diff
@@ -172,6 +172,10 @@ export interface ReviewWorkflowsFeature {
   options?: { numberOfWorkflows: number | null; stagesPerWorkflow: number | null };
 }
 
+export interface ContentReleasesFeature {
+  name: 'cms-content-releases';
```

Comment:
I think this is incomplete and should be something closer to. Technically a number should always be defined but I think it is possible that someone could make a mistake when configuring the license in which case it would be null.

```suggestion
  name: 'cms-content-releases';
  options?: {
  		maximumReleases: number | null;
  }
```